bioblend/galaxy/invocations/__init__.py

```diff
@@ -443,7 +455,7 @@ def wait_for_invocation(self, invocation_id, maxwait=12000, interval=3, check=Tr
                     f"Invocation {invocation_id} is still in non-terminal state {state} after {maxwait} s"
                 )
 
-    def _invocation_step_url(self, invocation_id, step_id):
+    def _invocation_step_url(self, invocation_id, step_id: str) -> str:
```

Comment:
```suggestion
    def _invocation_step_url(self, invocation_id: str, step_id: str) -> str:
```

Reply:
Oh, I missed the `invocation_id`.. Thanks!